components/brave_wallet/common/DEPS

```diff
@@ -1,9 +1,18 @@
 include_rules = [
   "+brave/third_party/argon2",
-  "+brave/third_party/bitcoin-core/src/src/base58.h",
 ]
 
 specific_include_rules = {
+  "bitcoin_utils\.cc": [
```

Comment:
is the escape necessary?

Reply:
Yep, so it has a meaning of 'exactly dot' instead of 'any character'

Reply:
could you also correct `brave_wallet_types.h` below?